crates/uv-toolchain/src/downloads.rs

```diff
@@ -0,0 +1,55 @@
+#[derive(Debug, PartialEq)]
+pub struct PythonDownloadMetadata {
+    name: ImplementationName,
+    arch: Arch,
+    os: Os,
+    libc: Libc,
+    major: i64,
+    minor: i64,
+    patch: i64,
+    url: &'static str,
+    sha256: Option<&'static str>,
+}
+
+#[derive(Debug, PartialEq)]
+pub(crate) enum Arch {
```

Comment:
Can we sync the names with https://github.com/astral-sh/uv/blob/c296da34bffd19d41b895bd5aa06e78d79c71bd4/crates/platform-tags/src/platform.rs#L75 and uses aliases where they don't match? So that we have consistent names across uv

Reply:
Yeah I plan on syncing with other crates, just didn't start with that yet. Thanks for the link.

Reply:
I'd like to address this in a follow-up so we can have these in a consistent location? I'm unsure what the best way to go about it is.

Reply:
Working on this in https://github.com/astral-sh/uv/pull/2918